polyform.md

```diff
@@ -14,7 +14,9 @@ The licensor grants you a copyright license for the software, to do everything y
 
 ## DL: Distribution License
 
-The licensor grants you an additional copyright license to distribute copies of the software, and if these terms grant you an additional license to make changes and new works, to distribute your changes and new works, as well.  You must ensure that anyone who gets a copy of any part of the software from you also gets the text of these terms or the URL for its text above.
+The licensor grants you an additional copyright license to distribute copies of the software.  If these terms grant you an additional license to make changes and new works, your license to distribute covers distributing your changes and new works, as well.  However, if these terms allow you to make changes and new works for internal business operations, your license to distribute does not cover distributing those changes and new works.
```

Comment:
Don't need the comma in `changes and new works, as well`

Reply:
Struck "as well" altogether in e38d80d78f3c8ebbc84e798018293a1705e7b5fd